docs/modules/sql/pages/mapping-to-jdbc.adoc

```diff
@@ -76,4 +86,100 @@ OPTIONS (
 ```
 
 <1> The name of the connector.
-<2> The name of the external data store configuration on your members.
\ No newline at end of file
+<2> The name of the external data store configuration on your members.
+
+== Data Type Mapping Between Hazelcast and MSSQL
```

Comment:
Could you add the same table for PostgreSQL and MySQL (there are not many differences there). Just to have it complete.